.github/workflows/main.yml

```diff
@@ -72,3 +70,87 @@ jobs:
         run: |
           ./devel/lint.sh  $( find . -name 'Dockerfile' \
           -o -name 'Dockerfile.*' )
+
+  build:
+    name: Build stage
+    needs: [lint-golang, lint-extra-files]
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup Go
+        uses: actions/setup-go@master
+        with:
+          go-version: 1.14
+        id: go
+
+      - name: Install dependencies
+        env:
+          # yamllint disable-line rule:line-length
+          INSTALL_KUSTOMIZE_URL: "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh"
+          # yamllint disable-line rule:line-length
+          OPERATOR_SDK_URL: "https://github.com/operator-framework/operator-sdk/releases/download/v1.0.0/operator-sdk-v1.0.0-x86_64-linux-gnu"
+          GO111MODULE: "on"
+        run: |
+          # GOPATH="$PWD/go"
+          # GOBIN="$GOPATH/bin"
+          # PATH="$GOBIN:$PATH"
+          curl -s "${INSTALL_KUSTOMIZE_URL}" | bash
+          go install -i
+          go get golang.org/x/lint/golint
+          go get github.com/kisielk/errcheck
+          go get github.com/golangci/golangci-lint/cmd/golangci-lint@v1.30.0
+          # curl --silent -L -o operator-sdk "${OPERATOR_SDK_URL}"
+          # chmod +x operator-sdk
+          # sudo mv operator-sdk /bin/
+
+      # https://docs.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets#using-encrypted-secrets-in-a-workflow
```

Comment:
Link is not dead, but it is stale.  It redirects to another page, but the `#anchor` is not valid.
This is the page it redirects to, but with a new anchor to appropriate section:
https://docs.github.com/en/free-pro-team@latest/actions/reference/encrypted-secrets#creating-encrypted-secrets-for-a-repository

Reply:
Thanks. Updated.